distribution/lib/Standard/Microsoft/0.0.0-dev/src/Internal/SQLServer_Dialect.enso

```diff
@@ -685,131 +706,69 @@ replace args metadata =
 ## PRIVATE
 make_extract_as_int enso_name sql_name=enso_name =
     Base_Generator.lift_unary_op enso_name arg->
-        as_int32 <| SQL_Builder.code "EXTRACT(" ++ sql_name ++ " FROM " ++ arg ++ ")"
+        SQL_Builder.code sql_name ++ "(" ++ arg ++ ")"
```

Comment:
After these changes this function is no longer `extract_as_int`, in fact if I'm not mistaken it is almost equivalent to `Base_Generator.make_function`, except that here the `lift_unary_op` will also check the number of arguments, whereas `make_function` does not do such a check. Still I'm not sure if this check is really needed, so IMHO we should replace it with `make_function`.

Reply:
Will fix